oat-sa/extension-tao-itemqti
# fix/AUT-1935/text-menu-not-fully-visible

**Related to:** [AUT-1935](https://oat-sa.atlassian.net/browse/AUT-1935)

**Description:**
Text in the Authoring Interaction panel is not fully visible.

**Changes:**

- Remove the truncate CSS effect.
- Reducing the size of the words that overflow the container box programmatically.

**How to check:**

- Check the Item interaction panel on Authoring to validate the labels show the full sentence.
- Try with another language such as French if you cannot see the issue in your current User language.

**TAE:** (Updated!) ✅ 
https://oat-sa.atlassian.net/browse/AUT-1935?focusedCommentId=180586
